File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/banner/BannerAdActivity.java
package com.applovin.enterprise.apps.demoapp.ads.max.banner;

import android.content.Intent;

import com.applovin.enterprise.apps.demoapp.ads.applovin.DemoMenuActivity;
import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem;

public class BannerAdActivity
        extends DemoMenuActivity
{
    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        DemoMenuItem[] result = {
                new DemoMenuItem( "Programmatic Banners", new Intent( this, ProgrammaticBannerAdActivity.class ) ),
                new DemoMenuItem( "Layout Editor Banners", new Intent( this, LayoutEditorBannerAdActivity.class ) ),
        };
        return result;
    }
}

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/banner/BannerAdActivity.kt
package com.applovin.enterprise.apps.demoapp.ads.max.banner

import android.content.Intent

import com.applovin.enterprise.apps.demoapp.ui.DemoMenuActivity
import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem

class BannerAdActivity : DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
        DemoMenuItem("Programmatic Banners", Intent(this, ProgrammaticBannerAdActivity::class.java)),
        DemoMenuItem("Layout Editor Banners", Intent(this, LayoutEditorBannerAdActivity::class.java)),
        DemoMenuItem("Jetpack Compose Banners", Intent(this, JetpackComposeBannerAdActivity::class.java)))
}
